src/core_modules/capture-core/metaDataStoreLoaders/baseLoader/quickStoreOperations/storeConstants.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import { quickStoreRecursively } from '../../IOUtils';
+import { getContext } from '../../context';
+
+export const storeConstants = (): Promise<void> => {
```

Comment:
Yeah basically this is is magic here!

Reply:
I was experimenting a bit. Let's have this discussion later and not force the annotations in this PR. Removing for consistency.